test/e2e/framework/kubelet/config.go

```diff
@@ -22,6 +22,8 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
```

Comment:
should maybe kill this file entirely and move the code to `test/e2e_node/services/kubelet.go`?